jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8487,6 +8487,9 @@ public String translateEscapes() {
 				case 'r':
 					builder.append('\r');
 					break;
+				case 's':
```

Comment:
Shouldn't this be guarded by `/*[IF Java14]*/`?

Reply:
Java 13 is no longer supported, and this code is only for Java 14 now.

Reply:
Right: I missed that the region was only for Java13+. Sorry for the distraction.